### PR TITLE
Flywheel 2022

### PR DESCRIPTION
**Pull request by vladaviedov**
The code has been tested and works
We don't know the final speed values yet

### CODE REVIEW

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -71,6 +73,9 @@
 	public static int spoolLiftRightMotor = 11;
 	public static int robotLiftLeftMotorPort = 9; 
 
+	public static int flywheelForwardPort = 5;
+	public static int flywheelReversePort = 4;
```
Comment (fielia) [inline src/main/java/frc/robot/RobotMap.java:77]: Unused values, I believe, now that we are using only one motor.

**Review by the-goat-man — CHANGES_REQUESTED** [review]
I think you're pretty much fine, other than that one thing.

File: src/main/java/frc/robot/RobotMap.java (hunk 1 above)
Comment (vladaviedov) [reply src/main/java/frc/robot/RobotMap.java]: Yep, looks like it was leftover from older commits

**Review by Oshawott101 — APPROVED** [review]

**Review by the-goat-man — APPROVED** [review]
did it with the wrong acct, my b

**Comment by vladaviedov** [comment]
Closing as the servo branch already was merged.

**Pull request closed without merging**